README.md

```diff
@@ -12,9 +12,9 @@ A collection of awesome useful and awesome links, resources and shiny things for
         * [Design](#design)
         * [eBooks](#ebooks)
         * [Geeky websites](#geeky-websites)
-            * [Challenges](#challenges)
-            * [CTF (Capture The Flag- Security Challenges)](#ctfcapture-the-flag--security-challenges)
-            * [Information Security (CTF & Hacking Platforms)](#information-security-ctf--hacking-platforms)
+        * [Challenges](#challenges)
+        * [CTF (Capture The Flag- Security Challenges)](#ctfcapture-the-flag--security-challenges)
+        * [Information Security (CTF & Hacking Platforms)](#information-security-ctf--hacking-platforms)
```

Comment:
These spaces are important to determine that these are subcategories of "Geeky Websites"

```suggestion
            * [Challenges](#challenges)
            * [CTF (Capture The Flag- Security Challenges)](#ctfcapture-the-flag--security-challenges)
            * [Information Security (CTF & Hacking Platforms)](#information-security-ctf--hacking-platforms)
```